.goreleaser.yaml

```diff
@@ -20,6 +20,21 @@ builds:
     ignore:
       - goos: linux
         goarch: '386'
+brews:
+  - name: idpbuilder
+    homepage: "https://cnoe.io"
+    repository:
```

Comment:
Do we want the repository for this to be `nimakaviani/homebrew-cnoe` or is this for testing? I would imagine maybe we'd want to consider `cnoe-io/tap` as the repository. It does seem that other big projects like argocd follow the pattern of `project/tap` for example argo uses `argoproj/tap` based on documentation here https://docs.brew.sh/Taps#the-brew-tap-command

Reply:
ah, an oversight with the owner name. fixed, thanks!

agreed. tap is a better name. renamed the repo to publish to tap.